docs/content/latest/quick-start/build-apps/go/ysql-pg-ssl.md

```diff
@@ -0,0 +1,335 @@
+    employee1 := &Employee{
+        Name:   "John",
+        Age:    35,
+        Language: []string{"Go"},
+    }
+    _, err = db.Model(employee1).Insert()
+    if err != nil {
+        log.Fatal(err)
+    }
+
+    _, err = db.Model(&Employee{
+        Name:      "Kelly",
+        Age:       35,
+        Language:  []string{"Golang", "Python"},
+    }).Insert()
+    if err != nil {
+        log.Fatal(err)
+    }
+
+    fmt.Println("Inserted data")
+
+    // Read from the table.
+    emp := new(Employee)
+    err = db.Model(emp).
+        Where("employee.id = ?", employee1.Id).
+        Select()
+    if err != nil {
+        log.Fatal(err)
+    }
+    fmt.Printf("Query for id=1 returned: ");
+    fmt.Println(emp)
+}
+```
+
+{{< note title="Note">}}
+If the `password` contains these special characters (#, %, ^), the driver may fail to parse the url. In such a case, use pg.Options() instead of pg.ParseURL() to initialize the Options in `ybsql_hello_world.go`. The standard PG environment variables except `PGPASSWORD` and `PGSSLROOTCERT` are implicitly read by the driver.
+{{< /note >}}
+
+```sh
+/* Modify the main() from the ybsql_hello_world.go script by replacing the first few lines and enabling pg.Options() */
+
+func main() {
+    opt := &pg.Options{
+        Password: os.Getenv("PGPASSWORD"),
+    }
+
+    CAFile := os.Getenv("PGSSLROOTCERT")
+    if (CAFile != "") {
+        CACert, err2 := ioutil.ReadFile(CAFile)
+        if err2 != nil {
+            log.Fatal(err2)
+        }
+
+        CACertPool := x509.NewCertPool()
+        CACertPool.AppendCertsFromPEM(CACert)
+
+        tlsConfig := &tls.Config{
+          RootCAs:            CACertPool,
+          ServerName:         host,
+        }
+        opt.TLSConfig = tlsConfig
+    }
+    db := pg.Connect(opt)
+
+    defer db.Close()
+
+    model := (*Employee)(nil)
+    err := db.Model(model).DropTable(&orm.DropTableOptions{
+        IfExists: true,
+    })
+    if err != nil {
+        log.Fatal(err)
+    }
+
+    err = db.Model(model).CreateTable(&orm.CreateTableOptions{
+        Temp: false,
+    })
+    if err != nil {
+        log.Fatal(err)
+    }
+
+    fmt.Println("Created table")
+
+    // Insert into the table.
+    employee1 := &Employee{
+        Name:   "John",
+        Age:    35,
+        Language: []string{"Go"},
+    }
+    _, err = db.Model(employee1).Insert()
+    if err != nil {
+        log.Fatal(err)
+    }
+
+    _, err = db.Model(&Employee{
+        Name:      "Kelly",
+        Age:       35,
+        Language:  []string{"Golang", "Python"},
+    }).Insert()
+    if err != nil {
+        log.Fatal(err)
+    }
+
+    fmt.Println("Inserted data")
+
+    // Read from the table.
+    emp := new(Employee)
+    err = db.Model(emp).
+        Where("employee.id = ?", employee1.Id).
+        Select()
+    if err != nil {
+        log.Fatal(err)
+    }
+    fmt.Printf("Query for id=1 returned: ");
+    fmt.Println(emp)
+}
+```
+
+## Disable SSL/TLS
```

Comment:
The ysql-pq-ssl version is I think better - it doesn't bother with this 'Disabled SSL/TSL section', and it doesn't appear to be needed here.

Reply:
This is because there is already a `ysql-pq.md` page without ssl setup and slightly different steps. But in case of the other two pages(ysql-pgx-ssl.md and ysql-pg-ssl), the user can choose to have ssl enabled or disabled and I felt like mentioning about the disabling steps on the same page rather than duplicating it on another page. Agreed that there is an inconsistency with the structure here though. Updates to the other pages may come in soon. Still waiting on some decisions from the engineering team.

Reply:
On second thoughts, I agree with you that we don't need to include it at all. I'll get rid of it. Thanks.